projects/material-css-vars/src/lib/_main.scss

```diff
@@ -26,19 +26,28 @@
     background: $mat-css-palette-background,
     foreground: $mat-css-palette-foreground,
   ));
+  // set global variable so passed-in content can use the theme
+  $mat-css-theme: $theme !global; 
 
   @at-root {
     // NOTE: light theme is the default theme
     @include angular-material-theme($theme);
+    @content;
 
     @if $dark-theme-selector {
+      $mat-css-theme: $dark-theme !global;
       #{$dark-theme-selector} {
         @include angular-material-theme($dark-theme);
+        // add content passed in, which can use the $theme variable to apply the dark theme to 
+        // other theme mixins needed by the app
+        @content; 
       }
     }
   }
 
   @include mat-css-other-overwrites;
+
+  $mat-css-theme: null !global; 
```

Comment:
Added this since your last comment too. We don't want the variable to still have a value if someone uses it outside the mixin